kms_log/main.tf

```diff
@@ -269,6 +299,10 @@ data "aws_iam_policy_document" "sqs_kms_cw_events_policy" {
         sid = "Allow SNS"
         effect = "Allow"
         actions = ["sqs:SendMessage"]
+        principals {
```

Comment:
There's something AWS doesn't like about this stanza, and the same stanza in "sqs_kms_es_events_policy". I don't see the problem, and the only error it will give is "Invalid value for the parameter Policy". Commenting out the principal makes it happy, as you can see in [my branch](https://github.com/18F/identity-terraform/compare/crissupb/kmslog...mark-from-usds/crissupb-kmslog-minor-fixes).

Reply:
Updated policies.  Cannot run deploy right now due to slow internet connection.